spec.md

```diff
@@ -1638,7 +1690,8 @@ will be rewritten as "the port `in`{.firrtl} is connected to the port
 ## Sub-fields
 
 The sub-field expression refers to a sub-element of an expression with a bundle
-type.
+type.  If the expression is of a constant bundle type, the sub-element shall be 
+of a constant type.
```

Comment:
I think there is a bug in the new wording. If I understand correctly, a bundle is const iff all fields are const. So if we take a subfield of a "mixed const" bundle:

```
wire w: {a: const UInt<8>, b: UInt<8>}
node a = w.a 
```

By the wording, wire w is non-const, and thus node a is also non-const.  I believe w.a should be const.

I'm not sure we needed to update the wording here at all. The type of a sub-field expression is the type of the field that was accessed, same as before.

Reply:
No, a bundle can be const regardless of its fields.  The wording on indexing operations says that indexing a const bundle will return a const type.

Reply:
In your example, a would be const since the type of the rhs is const.

Reply:
Examples added.